docs/ios/v2/how-to-guides/interact-with-room/room/polls.mdx

```diff
@@ -0,0 +1,155 @@
+---
+ title: Polls
+ nav: 7.26
+---
+
+## Introduction
+
+The Poll feature in the HMS SDK allows you to create, manage, and participate in polls and quizzes. Polls are an effective way to gather opinions, feedback, and insights from a group of people in a quick and convenient manner. With the Poll feature, you can create custom polls, set their properties, add questions, and manage responses. You can also receive real-time updates on poll status, and view poll results in an organized and easy-to-use format.
```

Comment:
would be good to add more detail on quizzes - perhaps in a separate short paragraph (for readability). something like: This feature can also be used to create quizzes. Under the hood, quizzes are like polls where questions where right or wrong answers.

Reply:
added